README.md

```diff
@@ -213,6 +213,28 @@ Make sure that the PR includes the changes made by running this command.
 git push --follow-tags
 ```
 
+### Publishing Pre-releases
+
+Read more in-depth [pre-release guides here](https://github.com/changesets/changesets/blob/main/docs/prereleases.md)
+
+Pre-releases let you publish an alpha/beta/next version of a component, allowing developers to test a component before fully releasing a component.
+
+Let's imagine we want to publish a `beta` release of some component. Our work is being done on a branch called `new-feature`
+
+1. Create a new branch off your component branch `git branch pre-release; git checkout pre-release`
```

Comment:
nit: you could also do `git checkout -b pre-release` but perhaps the above is easier to follow.

Reply:
+1 @shaneeza